File: frontend/src/app/models/menu-item-dto.ts
export interface MenuItemDto {
  menuItemId?: number;
  vendorId: number;
  categoryId: number;
  itemName: string;
  itemDescription?: string;
  price: number;
  calories?: number;
  isAvailable: boolean;
  createdAt?: Date;
  imageUrl?: string; // <-- add this property
}

File: backend/CateringNutrition/CateringNutrition.API/Dtos/menuitemsDTOS/MenuItemDto.cs
namespace CateringNutrition.API.Dtos.menuitemsDTOS
{
    public class MenuItemDto
    {

        public int VendorId { get; set; }
        public int CategoryId { get; set; }
        public string ItemName { get; set; }
        public string? ItemDescription { get; set; }
        public decimal Price { get; set; }
        public int? Calories { get; set; }
        public bool IsAvailable { get; set; } = true;
        public string? ImageUrl { get; set; }
    }
}
